index.html

```diff
@@ -214,6 +214,23 @@ <h3 class="section-title">Getting Data Into Your Widgets</h3>
 
         <script src="https://gist.github.com/davefp/6112780.js?file=curl2.sh"></script>
 
+        <h4>Widgets</h4>
+
+        <p>Dashing comes with few re-usable widgets (reminder: <i>data-view="WidgetName"</i> in dashboard defines the used widget):
```

Comment:
Also, I wasn't implying you had to be the one to make the updates -- just that these are the updates I recommend. Thanks!